.babelrc

```diff
@@ -0,0 +1,13 @@
+{
+  "presets": [
```

Comment:
trying `babel.config.json` instead of `.babelrc` in https://github.com/publiclab/PublicLab.Editor/pull/639